### PR TITLE
ES-356: Update CODEOWNERS file pre Freeze

### PR DESCRIPTION
**Pull request by ronanbrowne**
Update code owners file pre-freeze - these owners should get approval from product owners pre any merging.

### CODE REVIEW

**Review by driessamyn — DISMISSED** [review]

**Review by driessamyn — DISMISSED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,2 @@
...
-gradle/*           @corda/blt
-*.gradle           @corda/blt
-gradle.properties  @corda/corda5-team-leads
-
-*.kt               @corda/corda5-team-leads
-*.java             @corda/corda5-team-leads
-
-CODEOWNERS         @corda/blt @corda/corda5-team-leads
+# Reduced list for the duration of 5.0 code freeze 
+* @mnesbit @driessamyn @ronanbrowne @rick-r3 @fenryka
```
Comment (yarivus) [inline .github/CODEOWNERS]: Please include me too

Comment (ronanbrowne) [reply .github/CODEOWNERS]: Odd you are there on branch but PR isn't reflecting 

https://github.com/corda/corda-api/blob/ronanb/ES-356/update-code-owners/.github/CODEOWNERS#L2

Comment (ronanbrowne) [reply .github/CODEOWNERS]: Never mind there it is now

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Pull request merged**